src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,103 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_PROPERTY_SIZE = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(NUMBER_OF_FIGURES)) {
+            default:
+                return getCircle(FigureType.RANDOM);
+            case 0:
+                return getRectangle(FigureType.RANDOM);
+            case 1:
+                return getRightTriangle(FigureType.RANDOM);
+            case 2:
+                return getSquare(FigureType.RANDOM);
+            case 3:
+                return getIsoscelesTrapezoid(FigureType.RANDOM);
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        switch (random.nextInt(NUMBER_OF_FIGURES)) {
+            default:
+                return getCircle(FigureType.DEFAULT);
+            case 0:
+                return getRectangle(FigureType.DEFAULT);
+            case 1:
+                return getRightTriangle(FigureType.DEFAULT);
+            case 2:
+                return getSquare(FigureType.DEFAULT);
+            case 3:
+                return getIsoscelesTrapezoid(FigureType.DEFAULT);
+        }
```

Comment:
Read the task, `getDefaultFigure` should return `Circle` with color `WHITE` and radius `10`